special-use/special-use-suds-mapping-get.json

```diff
@@ -0,0 +1,82 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Special use application",
+    "description": "A rough sketch of the JSON response to a GET HTTP request for a special use permit from our ePermitting system for the U.S. Forest Service",
+    "definitions": {
+      "applicant-info": {
+        "type": "object",
+        "properties": {
+          "contactControlNumber": { "type": "string", "basic-api-field" : "CONT_CN"},
+      		"firstName": { "type": "string", "basic-api-field" : "FIRST_NAME"},
+      		"lastName": {	"type": "string", "basic-api-field" : "LAST_NAME" },
+          "dayPhone": { "$ref": "#/definitions/phone-number" },
+          "eveningPhone": { "$ref": "#/definitions/phone-number" },
+          "emailAddress": { "type": "string", "basic-api-field" : "EMAIL"},
+          "mailingAddress": { "type": "string", "basic-api-field" : "ADDRESS_1"},
+          "mailingAddress2": { "type": "string", "basic-api-field" : "ADDRESS_2"},
+          "mailingCity": { "type": "string", "basic-api-field" : "CITY_NAME"},
+          "mailingState": { "type": "string", "basic-api-field" : "STATE_CODE"},
+          "mailingZIP": { "type": "integer", "basic-api-field": "POSTAL_CODE"},
+      		"organizationName": {
+      			"description": "The name of the group, company, organization or event applying",
+      			"type": "integer",
+      		},
+          "website": { "type": "string", "suds-table": "Contact", "suds-field": "Remarks" },
+          "orgType": { "type": { "enum": [ "Individual"  , "Corporation"  , "Limited Liability Company"  , "Partnership or Association"  , "State Government or Agency"  , "Local Government or Agency"  , "Nonprofit" ] }, "basic-api-field" : "ORG_TYPE, ORG_CODE" },
+        "required": ["firstName","lastName","dayPhone","emailAddress","mailingAddress","mailingCity","mailingZIP","mailingState"]
+      },
+      "phoneNumber": {
+        "type": "object",
+        "properties": {
+          "areaCode": { "type": "integer", "basic-api-field": "AREA_CODE"},
+          "number": { "type": "integer", "basic-api-field": "PHONE_NUMBER"},
+          "extension": { "type": "integer", "basic-api-field": "EXTENSION"},
+          "type": { "basic-api-field" : "PHONE_NUMBER_TYPE"},
+        },
+        "required": ["areaCode","number","type"]
+      },
+      "non-commercial-fields": {
+        "type": "object",
+        "properties": {
+      		"activityDescription": { "type": "string", "note": "See the 'AUTOPOPULATED_BY_MIDDLEWARE_NOT_INTAKE_MODULE' note."},
```

Comment:
Do we have a sense of how this would be parsed by the `get`? or is that for NCI to determine. This comment is probably for all the autopopulated fields.